spec/mailers/checkout_mailer_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+RSpec.describe CheckoutMailer, type: :mailer do
+  let(:user) { User.create( full_name: 'Lucas', email: 'lucas@email.com', password: "test" )}
```

Comment:
Line is too long. [93/80]<br>Space missing inside }.<br>Space inside parentheses detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.